### PR TITLE
Update Biome Text Labels and Variable Name

### PR DESCRIPTION
**Pull request by bennybroseph**
Updates the new Text which displays the Biome and Wave count variable to a proper name.
Updates Final Destination to ??? to keep the name an easter egg rather than a blatant label.

### CODE REVIEW

File: src/battle-scene.ts [hunk 1]
```diff
@@ -1244,12 +1245,13 @@ export default class BattleScene extends SceneBase {
     });
   }
 
-  updateWaveCountText(): void {
+  updateLineOneText(): void {
```
Comment (TempsRay) [inline src/battle-scene.ts:1248]: This is still `LineOneText` in the green?

Comment (bennybroseph) [reply src/battle-scene.ts:1248]: bro chill lol

File: src/data/biomes.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export function getBiomeName(biome: Biome | -1) {
   case Biome.SPACE:
     return "Stratosphere";
```
Comment (TempsRay) [inline src/data/biomes.ts:22]: I would also like this renamed so it matches the biome name too

**Review by TempsRay — COMMENTED** [review]
Also update commit title to mention renaming biome names

**Review by TempsRay — APPROVED** [review]

**Pull request merged**